assets/blueprints/blueprint.json

```diff
@@ -1,18 +1,20 @@
 {
 	"$schema": "https://playground.wordpress.net/blueprint-schema.json",
-	"landingPage": "/wp-admin/profile.php",
+	"landingPage": "\/wp-admin\/profile.php",
 	"preferredVersions": {
 		"php": "7.4",
 		"wp": "latest"
 	},
 	"steps": [
 		{
-			"step": "login"
+			"step": "login",
+      "username": "admin",
+			"password": "password"
 		},
 		{
 			"step": "installPlugin",
-			"pluginData": {
-				"resource": "wordpress.org/plugins",
+			"pluginZipFile": {
```

Comment:
@jeffpaul The documentation on the site is old. The latest schema can be seen here https://playground.wordpress.net/blueprint-schema.json

For example, the `pluginZipFile` is deprecated:

<img width="895" alt="schema" src="https://github.com/user-attachments/assets/3f3d36de-332a-4db7-9d3f-88ca73285c47" />

VS Code should be able to autocomplete and suggest the field values per https://code.visualstudio.com/docs/languages/json#_json-schemas-and-settings